FAQ: default username/password and password reset (#85)
audiobookshelf/audiobookshelf-web: content/faq/2.server.md

If you don't provide mounts, the container cannot see anything outside of the container to store your data or see your books.

There are also volume mounts, but this only discusses bind mounts.

# What is the default admin username and password?

There is not a default username and password.
You will set the root username and password when you first visit the server in a web browser.
Once you have created this first user, you can create other admin and user accounts.

# How can I reset a password?

A user password can be reset for an individual account by an admin account on the User page of the server settings.
If the user is using OIDC to log in, the password should be reset in the OIDC provider.

> If you are not able to access an admin account to reset a password or have forgotten and admin account password, you can manually edit the SQLite database to clear the `pash` column for the user.

Reviewer: ```suggestion
If you need to reset the password for the root user you will need to manually edit the SQLite database. In the `users` table you will need to find the root user row and update the `pash` column to NULL.
```

> If you need to reset the password for the root user you will need to manually edit the SQLite database. In the `users` table you will need to find the root user row and update the `pash` column to NULL.

Reviewer: ```suggestion
If you forgot the password for the root user you will need to manually edit the SQLite database. In the `users` table find the root user row and update the `pash` column to NULL.
```

> This will allow you to log in with a blank password so you can set the password on that user.

Reviewer: ```suggestion
This will allow you to log in as the root user with a blank password and you can set a new password after logging in.
```
